src/AutoMapper.Collection.Tests/MapCollectionWithEqualityTests.cs

```diff
@@ -135,14 +135,14 @@ public void Should_Be_Fast_With_Large_Lists_MultiProperty_Mapping_Cant_Extract_N
             {
                 x.AddCollectionMappers();
                 // ReSharper disable once NegativeEqualityExpression
-                x.CreateMap<ThingDto, Thing>().EqualityComparison((ThingDto dto, Thing entity) => dto.ID == entity.ID && !(dto.ID != entity.ID));
+                x.CreateMap<ThingDto, Thing>().EqualityComparison((ThingDto dto, Thing entity) => dto.ID == entity.ID && dto.ID == entity.ID);
```

Comment:
Regarding to the comments and the name of the test this expression should be double negative.

Reply:
in French a double-negative shows emphasis, but we are in logic-land with simple 0/1 or F/T (OK we'll ignore the database NULL where folk can't make up their minds and Prof Hoare said it was worst invention ever, but at least C# team are mastering the #nullable enable/disable!).
If that is the only objection I still say should comply with what the analyzers recommend as more devs are getting theirs brains in tune with "modern" coding phraseology like expression bodies and ??throw constructs).
Personally I am aghast at "we don't have a standard" and would strongly counsel use of .editorconfig for consistency, but I'm not a complete zealot, and agree that readability trumps minimisation so I'll just have to biden my time until you agree to promote this PR with/out further "rights" amendments.

Reply:
From my view it is good to make the cleanup but it should be good if you also can add an editorconfig file so that the style will be the same for all contributors,

For the tests, even if you have a standard you may need to not use that for some reason, example is in test-code to ensure that also if you use use code with the best syntax still is working. Don't remember exactly when this test was introduced but without that we was having a bug inside some of the expression visitors that are used.